README.md

```diff
@@ -138,6 +138,21 @@ The B3 codec assumes it will receive lowercase HTTP headers, as this seems
 to be the standard in the popular frameworks like Flask and Django.
 Please make sure your framework does the same.
 
+## Prometheus metrics
```

Comment:
I think it's better to move this to L83 as level-4 header, i.e. as a subsection of Configuration/Production.

Reply:
moved